collector/CMakeLists.txt

```diff
@@ -93,6 +93,11 @@ add_test(collector-tests runUnitTests)
 # Falco Wrapper Library
 set(BUILD_DRIVER OFF CACHE BOOL "Build the driver on Linux" FORCE)
 set(USE_BUNDLED_DEPS OFF CACHE BOOL "Enable bundled dependencies instead of using the system ones" FORCE)
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
+	set(USE_BUNDLED_LUAJIT OFF CACHE BOOL "Disable building of the bundled LuaJIT" FORCE)
+else()
+	set(USE_BUNDLED_LUAJIT ON CACHE BOOL "Enable building of the bundled LuaJIT" FORCE)
+endif()
```

Comment:
Hard to tell from the context; does this change the default on x86?  Where is this variable used?

Reply:
Done